client/src/components/ComparisonModal.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+
+const ComparisonModal = ({ isVisible, hideModal }) => {
```

Comment:
I haven't used `createPortal` before, but I don't understand how this component receives data. Is it meant to receive data yet, or are these all just placeholder elements for styling?